backend/app/core/config.py

```
@@ -0,0 +1,30 @@
+
+    # App settings
+    app_name: str = "DOCX Translation API"
+    app_version: str = "0.1.0"
+    debug: bool = False
+
+    # API settings
+    max_file_size: int = 50 * 1024 * 1024  # 50MB
+    supported_formats: list = [".docx"]
+
+    # Security
+    secret_key: str = "your-secret-key-here"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "secret_key"

(S105)